src/duckling/entitysystem/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './attribute.component';
+export * from './attribute-component.service';
+export * from './base-attribute-service';
+export * from './entity';
+export * from './entity.component'
+export * from './entity-system';
```

Comment:
entity-system should be deleted because it does nothing, remove the reference to it here